packages/nc-gui/components/project/AccessSettings.vue

```diff
@@ -85,7 +85,7 @@ const updateCollaborator = async (collab: any, roles: ProjectRoles) => {
         WorkspaceRolesToProjectRoles[currentCollaborator.workspace_roles as WorkspaceUserRoles] === roles &&
         isEeUI)
     ) {
-      await removeProjectUser(activeProjectId.value!, collab)
+      await removeProjectUser(activeProjectId.value!, currentCollaborator as any)
```

Comment:
fix the type instead of using `any`. (apply to all)

Reply:
Can I use `as unknown as User` to remove ts error because `removeProjectUser/updateProjectUser/createProjectUser` accept type `User` and current type is `Collaborators`

Reply:
For now I added `as unknown as User` let me know if this is not a correct way

Reply:
I'm fine with it